sinks/cortex/cortex.go

```diff
@@ -210,20 +210,33 @@ func makeWriteRequest(metrics []samplers.InterMetric, tags map[string]string) *p
 // legal set of characters (see https://prometheus.io/docs/practices/naming)
 // and we drop tags which are not in "key:value" format
 // (see https://prometheus.io/docs/concepts/data_model/)
+// we also take "last value wins" approach to duplicate labels inside a metric
```

Comment:
I think it's good we've clarified "last write wins" within a datapoint itself - but it looks like `tags` takes precedence over the label(s) on a datapoint. Is this intentional? Do we want it to be the other way around? Can we document the decision we go with?

Reply:
Yep, iirc `tags` is something either manually supplied by the user configuring veneur and as such should take precedence imo. Do you agree? I can add it to docs in the comment and the README

Reply:
I think that's fine. I could see an argument for either way, but I think this approach may lead to less confusion by default ("why does this standardized tag have a non-standardized value")

Reply:
+1